config/nyc.json

```diff
@@ -1,5 +1,6 @@
 {
   "reporter": ["lcov", "text"],
   "extension": [".ts"],
-  "all": true
+  "all": true,
+  "cache": false
```

Comment:
Did this achieve what it should? Just to reassure we are not unnecessarily sacrifice on performance here.

Reply:
didn't seem to make any difference, but i think it's a good change for us, because from nyc's [caching](https://github.com/istanbuljs/nyc#caching) information it says that it stores the cache in `<project-root>/node_modules/.cache/nyc` and github actions cache stores the whole node_modules folder, so this should help keep differing ci runs to not use the same cache in case the files have changed